include/xlnt/cell/cell.hpp

```diff
@@ -365,7 +369,8 @@ class XLNT_API cell
     void number_format(const class number_format &format);
 
     /// <summary>
-    /// Returns the font applied to the text in this cell.
+    /// Returns a copy of the font applied to the text in this cell. If no font has been set (format().has_font() returns false),
+    /// a default-constructed font will be returned.
```

Comment:
I'm not against this behavior, but I think this is not exactly the current behavior. It is implemented as `format().font()`. `format::font()` does not throw an exception anymore, but `cell::format` may (if no format is set).